bindings_generator/src/api.rs

```diff
@@ -406,9 +411,10 @@ impl Ty {
                 }
             }
             ty => {
-                let module = format_ident!("{}", module_name_from_class_name(ty));
+                //let module = format_ident!("{}", module_name_from_class_name(ty));
                 let ty = format_ident!("{}", ty);
-                Ty::Object(syn::parse_quote! { crate::generated::#module::#ty })
+                //Ty::Object(syn::parse_quote! { crate::generated::#module::#ty })
+                Ty::Object(syn::parse_quote! { crate::generated::#ty })
             }
```

Comment:
I don't think it's necessary to leave the old code commented out here. We have `git blame` for peering into the history of a file. Commented but unexplained code are liable to become confusing down the line.